rai/models.go

```diff
@@ -255,7 +255,7 @@ type TransactionAsyncFile struct {
 
 type ArrowRelation struct {
 	RelationID string
-	Table      interface{}
+	Table      []interface{}
```

Comment:
is there a way we can avoid `interface{}` altogether? Are the list items not all of the same type? If not, maybe at least put a comment here explaining what types they are?

Reply:
thanks @vilterp, yep the list items should have the same type but we can't decide what's data type until we get the metadata. This should be enhanced once we start constructing results based on the metadata :pray: